driver-nats-streaming/pom.xml

```diff
@@ -24,10 +24,15 @@
 
     <artifactId>driver-nats-streaming</artifactId>
     <dependencies>
+       <dependency>
+          <groupId>com.google.guava</groupId>
+          <artifactId>guava</artifactId>
+          <version>29.0-jre</version>
+        </dependency>
         <dependency>
             <groupId>io.nats</groupId>
             <artifactId>java-nats-streaming</artifactId>
-            <version>2.1.0</version>
+            <version>2.2.0</version>
```

Comment:
since upstream openmessaging-benchmark is at `2.1.0`, to be more conservative, we could just go to the latest patch version `2.1.6` instead of going to `2.2.0`: https://github.com/nats-io/stan.java/compare/2.1.0...2.1.6#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R55